test/functional/bipdersig-p2p.py

```diff
@@ -46,12 +46,8 @@ class BIP66Test(ComparisonTestFramework):
     def __init__(self):
         super().__init__()
         self.num_nodes = 1
-
-    def setup_network(self):
         # Must set the blockversion for this test
-        self.nodes = start_nodes(self.num_nodes, self.options.tmpdir,
-                                 extra_args=[['-whitelist=127.0.0.1', '-blockversion=2']],
-                                 binary=[self.options.testbinary])
+        self.extra_args = [[], ['-whitelist=127.0.0.1', '-blockversion=2']]
```

Comment:
It turns out the comment was wrong (and had presumably been copy-pasted from another test). blockversion is not required in this test, since the bitcoin node is not generating any blocks. I've removed the extra_args entirely.

I've also added your suggested asserts.